doc/Type/Any.pod6

```diff
@@ -115,7 +115,7 @@ scalar in a list context by calling C<.list> and turning it into an C<Array>.
 
 Defined as:
 
-    method push(|values --> Positional:D)
+    method push(Any:U \SELF: |values --> Positional:D)
```

Comment:
This is not related to the rest of the PR. On top of all, I really *don't* like PRs that involve 80 files. If you want PRs to be reviewed, please do not make more than a bunch of changes, on a (small) bunch of files or a single file. Since you've changed the definition, you might want to revise or change the explanation, too.